cmd/kubeadm/app/cmd/phases/join/controlplaneprepare.go

```diff
@@ -163,27 +163,3 @@ func runControlPlanePrepareKubeconfigPhaseLocal(c workflow.RunData) error {
 
 	return nil
 }
-
-func runControlPlanePrepareJoinSubphase(component string) func(c workflow.RunData) error {
-	return func(c workflow.RunData) error {
-		data, ok := c.(controlPlanePrepareData)
-		if !ok {
-			return errors.New("control-plane-prepare phase invoked with an invalid data struct")
-		}
-
-		// Skip if this is not a control plane
-		if data.Cfg().ControlPlane == nil {
-			return nil
-		}
-
-		cfg, err := data.InitCfg()
-		if err != nil {
-			return err
-		}
-
-		// Creates static pod manifests file for the control plane components to be deployed on this node
-		// Static pods will be created and managed by the kubelet as soon as it starts
-		fmt.Printf("[control-plane-prepare] Creating static Pod manifest for %q\n", component)
-		return controlplane.CreateStaticPodFiles(kubeadmconstants.GetStaticPodDirectory(), &cfg.ClusterConfiguration, &cfg.LocalAPIEndpoint, component)
```

Comment:
by removing this function, in this same file please rename:
`runControlPlaneSubphase` to `runControlPlanePrepareManifestsSubphase`

Reply:
done